python/ray/serve/batching.py

```diff
@@ -13,13 +13,22 @@
 from ray.util.annotations import PublicAPI
 
 
+END_ASYNC_ITER_TOKEN = None
+
+
 @dataclass
 class _SingleRequest:
     self_arg: Optional[Any]
     flattened_args: List[Any]
     future: asyncio.Future
 
 
+@dataclass
+class _GeneratorIterResponse:
+    result: Optional[Any]
```

Comment:
```suggestion
    result: Any
```
nit (these are equivalent)